antora-playbook.yml

```diff
@@ -96,12 +96,13 @@ asciidoc:
     prod-operator-image-name: che-operator
     prod-prev-ver-major: 6
     prod-prev-ver: 7.18
+    che-ver: 7.20.2
     prod-short: Che
     prod-upstream: Eclipse{nbsp}Che
     prod-url-secure: \https://che-host
     prod-url: \https://che-host:che-port
     prod-ver-major: 7
-    prod-ver: 7.19
+    prod-ver: 7.20
```

Comment:
Latest release is 7.21, and latest che-ver is https://github.com/eclipse/che-operator/tree/master/olm/eclipse-che-preview-openshift/deploy/olm-catalog/eclipse-che-preview-openshift/7.21.1

Reply:
Updating